### PR TITLE
Implementation of getting the iommu pci devices address

### PR DESCRIPTION
**Pull request by iccaszhulili**
Add the function of getting the pci devices addresses
sharing the same iommuGroup.

Signed-off-by: Lily Zhu <lizhu@redhat.com>

### CODE REVIEW

File: virttest/libvirt_xml/nodedev_xml.py [hunk 1]
```diff
@@ -140,8 +140,27 @@ class for capability whose type is pci.
     #  <numa node='0'/>
     #</capability>
 
+    # or the Example of iommu:
+    #<capability type="pci">
```
Comment (chunfuwen) [inline virttest/libvirt_xml/nodedev_xml.py]: follow the comment format(leave one white space between # and following words):# xx ,and the following have similar issues.

Comment (iccaszhulili) [reply virttest/libvirt_xml/nodedev_xml.py]: Updated

File: virttest/libvirt_xml/nodedev_xml.py [hunk 2]
```diff
@@ -162,6 +181,10 @@ def __init__(self, virsh_instance=base.virsh):
                                  parent_xpath='/capability',
                                  marshal_from=self.marshal_from_address,
                                  marshal_to=self.marshal_to_address)
+        accessors.XMLElementList('iommuGroup_number', self,
+                                 parent_xpath='/iommuGroup',
+                                 marshal_from=self.marshal_from_address,
+                                 marshal_to=self.marshal_to_address)
```
Comment (kylazhang) [inline virttest/libvirt_xml/nodedev_xml.py]: The iommuGroup_number should be an attribute? And here should be the 'iommuGroup_address' ?

Comment (iccaszhulili) [reply virttest/libvirt_xml/nodedev_xml.py]: @kylazhang Updated, please have a check

**Pull request merged**